main.go

```diff
@@ -104,7 +109,7 @@ func parseVariables(cmd *cobra.Command, args []string) {
 			SkipEmptyColumns: true,
 		}
 
-		templateCSVVariables[key], err = csvp.Untyped(csvBytes)
+		templateCSVVariables[key], err = csvp.Typed(csvBytes)
```

Comment:
Isnt this a big change here?Does this need to be part of this patch?

Reply:
The unit tests didn't pass because of it, we clearly use typed CSVs, no idea how it was in master